source/protocol/lsp-messages.ads

```diff
@@ -4204,6 +4205,7 @@ package LSP.Messages is
    --	 * @since 3.15.0
    --	 */
    --	serverInfo?: {
+
```

Comment:
Could you drop this an extra empty line to keep original comments intact?

Reply:
Ok